### PR TITLE
Popover above the button

### PR DESCRIPTION
**Pull request by ellynliu**
Task: task-992210

Link: [preview](http://localhost:1111/)

[Explain the context of this pull request here]

## Testing

1. http://localhost:1111/components/popover.html

## Additional information

[Optional]

## Contributor checklist

- [x] Did you update the description of this pull request with a review link and test steps?
- [x] Did you submit a changeset? Changesets are required for all code changes.
- [ ] Does your pull request implement complex UI logic (js/ts)? Consider adding an integration test to test your user flow.
- [ ] Does your pull request add a new page to the documentation site? Add your new page for automated accessibility testing in /integration/tests/accessibility.

### CODE REVIEW

File: site/src/components/popover.md [hunk 1]
```diff
@@ -75,3 +75,12 @@ Popover is left aligned by default, but also can be centered or right aligned re
 	<div class="popover-content">Popover content.</div>
 </details>
 ```
+
```
Comment (ollips) [inline site/src/components/popover.md:78]: Can you add a heading here to separate horizontal and vertical alignment please?

```
### Vertical alignment
```

Comment (ollips) [reply site/src/components/popover.md:78]: And you can also add a section mentioning horizontal and vertical can be mixed up

Comment (ellynliu) [reply site/src/components/popover.md:78]: added

File: site/src/components/popover.md [hunk 2]
```diff
@@ -75,3 +75,24 @@ Popover is left aligned by default, but also can be centered or right aligned re
...
+The popover can also show up above the button.
+
+```html
+<details class="popover popover-top margin-xxs">
+	<summary class="button">Popover on top of the button</summary>
+	<div class="popover-content">Popover content.</div>
+</details>
+
+<details class="popover popover-top popover-center margin-xxs">
```
Comment (ollips) [inline site/src/components/popover.md:89]: Can you split that example into two and say "vertical and horizontal alignment can be used together" in between

File: site/src/components/popover.md [hunk 3]
```diff
@@ -75,3 +75,24 @@ Popover is left aligned by default, but also can be centered or right aligned re
...
 </details>
 ```
+
+### Vertical alignment
+
+The popover can also show up above the button.
+
+```html
+<details class="popover popover-top margin-xxs">
+	<summary class="button">Popover on top of the button</summary>
```
Comment (ollips) [inline site/src/components/popover.md:85]: Popover on top of the button -> `Popover on top`

File: site/src/components/popover.md (hunk 2 above)
Comment (ellynliu) [reply site/src/components/popover.md:98]: added

**Comment by wibjorn** [comment]
Needs a minor changeset for both css and site

**Review by wibjorn — APPROVED** [review]

**Review by ollips — APPROVED** [review]

**Pull request merged**